frame/benchmarking/src/tests.rs

```diff
@@ -105,9 +102,6 @@ impl frame_system::Trait for Test {
 
 impl Trait for Test {
 	type Event = ();
-	type BlockNumber = u32;
-	type Origin = Origin;
-	type AccountId = u64;
```

Comment:
I didnt understand why these types were used instead of the ones from `frame_system`, so I updated it to what I would have expected to see.